buildtools/windows-dev-environment-notes.md

```
@@ -1,59 +1,44 @@
+The minimum system requirements for compiling the Tari suite are as follows:
+- 64-bit processor
+- 8gb RAM minimum
+- 4gb download bandwidth for the various packages.
+- A minimum of 40gb free space
 
-Need to have ```winget``` installed and working, which requires ```App Installer```, not only installed, but updated to the latest version.
+This guide utilises the PowerShell command line to install the dependencies.
 
-Using Microsoft Edge, open the following URL:
+You will require the following package managers to set up the development environment:
+* winget
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)